.github/CODEOWNERS

```
@@ -17,89 +17,137 @@
+/data/changelogs/stream.yaml @tsmith512 @dcpena @cloudflare/pcx-technical-writing
+/content/workers/ @cloudflare/workers-docs @GregBrimble @WalshyDev @deadlypants1973 @cloudflare/pcx-technical-writing
+/data/changelogs/workers.yaml @cloudflare/workers-docs @GregBrimble @WalshyDev @deadlypants1973 @cloudflare/pcx-technical-writing @irvinebroque @admah
 /content/zaraz/ @bjesus @jonnyparris @cloudflare/pcx-technical-writing
 /data/changelogs/zaraz.yaml @bjesus @jonnyparris @cloudflare/pcx-technical-writing
-/content/workers/runtime-apis/ @irvinebroque @jasnell @cloudflare/pcx-technical-writing
-/content/workers/platform/ @irvinebroque @tanushreesharma-cf @cloudflare/pcx-technical-writing
-/content/workers/configuration/bindings/ @irvinebroque @cloudflare/pcx-technical-writing
-/content/workers/configuration/compatibility-dates/ @irvinebroque @cloudflare/pcx-technical-writing
-/content/workers/reference/migrate-to-module-workers/ @irvinebroque @cloudflare/pcx-technical-writing
-/content/workers/reference/security-model/ @irvinebroque @cloudflare/pcx-technical-writing
-/content/workers/_partials/_platform-compatibility-dates/ @irvinebroque @kflansburg @cloudflare/pcx-technical-writing
-/content/pages/framework-guides/ @igorminar @dario-piotrowicz @cloudflare/pcx-technical-writing
-
-# Firewall Rules & WAF
-/content/firewall/ @cloudflare/firewall @cloudflare/pcx-technical-writing
-/content/waf/ @cloudflare/firewall @cloudflare/pcx-technical-writing
-/content/waf/change-log/ @cloudflare/firewall @unknownhad @vs-mg @cloudflare/pcx-technical-writing
-/data/changelogs/waf.yaml @cloudflare/firewall @unknownhad @vs-mg @cloudflare/pcx-technical-writing
+/content/workers/runtime-apis/ @irvinebroque @jasnell @deadlypants1973 @cloudflare/pcx-technical-writing
+/content/workers/platform/ @irvinebroque @tanushreesharma-cf @deadlypants1973 @cloudflare/pcx-technical-writing
+/content/workers/configuration/bindings/ @irvinebroque @deadlypants1973 @cloudflare/pcx-technical-writing
+/content/workers/configuration/compatibility-dates/ @irvinebroque @deadlypants1973 @cloudflare/pcx-technical-writing
+/content/workers/reference/migrate-to-module-workers/ @irvinebroque @deadlypants1973 @cloudflare/pcx-technical-writing
+/content/workers/reference/security-model/ @irvinebroque @deadlypants1973 @cloudflare/pcx-technical-writing
+/content/workers/_partials/_platform-compatibility-dates/ @irvinebroque @kflansburg @deadlypants1973 @cloudflare/pcx-technical-writing
+/content/pages/framework-guides/ @igorminar @dario-piotrowicz @deadlypants1973 @cloudflare/pcx-technical-writing
 
 # DDoS Protection
-/content/ddos-protection/change-log/ @jdesgats @antoinecordelle @cloudflare/pcx-technical-writing
-/data/changelogs/ddos-http.yaml @jdesgats @antoinecordelle @cloudflare/pcx-technical-writing
-/data/changelogs/ddos-network.yaml @jdesgats @antoinecordelle @cloudflare/pcx-technical-writing
+/content/ddos-protection @pedrosousa @cloudflare/pcx-technical-writing
```

```suggestion
/content/ddos-protection/ @pedrosousa @cloudflare/pcx-technical-writing
```
For the sake of consistency.